hrm-domain/packages/hrm-types/ConversationMedia.ts

```diff
@@ -107,9 +107,20 @@ export type ExportTranscriptMessage = {
 };
 
 export type ExportTranscript = {
-  accountSid: HrmAccountId;
+  accountSid: string;
   serviceSid: string;
   channelSid: string;
   messages: ExportTranscriptMessage[];
   participants: ExportTranscripParticipants;
 };
+
+export type ExportTranscriptDocument = {
+  accountSid: HrmAccountId;
```

Comment:
This would be an `AccountSID` type being picky. They are both practically the same at this point though so meh